modules/odc-deploying-java-applications.adoc

```diff
@@ -26,7 +26,7 @@ Use the following instructions to upload a JAR file in the *Topology* view to de
 
 . In the *Name* field, enter a unique component name to name the associated resources.
```

Comment:
**Suggestion**: The word "name" has been used too many times in a relatively short sentence. Consider restructuring the sentence for better readability and aesthetics. For example:

In the **Name** field, enter a unique component name to indicate the associated resources.

OR

In the **Name** field, enter a unique component to name the associated resources.

----------------------------------
Another generic advice would be to restructure such sentences in the following format:

_To <achieve some objective>, <perform some action>._

I understand that these bullet points follow other sentences that are beyond the scope of your PR, and hence the suggestion for the future.